spinner/spinner.go

```diff
@@ -0,0 +1,15 @@
+package spinner
```

Comment:
Drop in favor of in-lining simple spinners where they're needed and cleaning them up immediately.